src/libs/ReportUtils.ts

```diff
@@ -2733,6 +2733,11 @@ function canEditMoneyRequest(reportAction: OnyxInputOrEntry<ReportAction<typeof
         return false;
     }
 
+    const transaction = linkedTransaction ?? getLinkedTransaction(reportAction ?? undefined);
+    if (!transaction?.transactionID || (transaction?.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD && !isEmptyObject(transaction.errors))) {
```

Comment:
NAB: It would capture the intent better and be  easier to read if we extract the if statement condition into a variable called `transactionFailedToCreate`. Alternatively we could explain with a comment.